src/main.rs

```diff
@@ -140,6 +142,14 @@ async fn handle_roadmap(ctx: &Context, message: &Message) -> anyhow::Result<()>
     Ok(())
 }
 
+async fn handle_ask(ctx: &Context, message: &Message) -> anyhow::Result<()> {
+    let response = respond_ask().await?;
+
+    reply_chunked(ctx, message.author.mention(), message.channel_id, response).await?;
+
+    Ok(())
+}
+
```

Comment:
```suggestion
async fn handle_ask(ctx: &Context, message: &Message) -> anyhow::Result<()> {
    reply_chunked(
        ctx,
        message.author.mention(),
        message.channel_id,
        "Don't ask to ask, just ask! \nhttps://dontasktoask.com/".to_string(),
    )
    .await
}
```

There's a few points here; 
1. We don't need to create a function that's only going to return a string. 
2. We can return the Result from `reply_chunked` - we don't need to use `?` to check the result and then return the Ok.